modules/tinymce/src/plugins/fullpage/main/ts/core/Parser.ts

```diff
@@ -12,6 +12,23 @@ import HtmlSerializer from 'tinymce/core/api/html/Serializer';
 import Tools from 'tinymce/core/api/util/Tools';
 import * as Settings from '../api/Settings';
 
+interface Data {
```

Comment:
Great work figuring this out

Reply:
Tbh I just kept adding things until the types no longer complained, so was largely automated.